modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -69,7 +69,12 @@ import WindowManager from './WindowManager';
  * ed.render();
  */
 
-export interface EditorUi {
+export interface EditorUiApi {
```

Comment:
Would this be better in the `Ui` module instead of here? I'm honestly undecided, so just asking to get others thoughts.

Reply:
Just remember this also needs the moxiedocs setup for this new API... paging @tylerkelly13.

Reply:
Those docs will need to be added here: https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js

- [ ] Add API doc strings

Reply:
@SimonFc, Re: API Docs. if you're not sure how to do this, I can provide guidance. Would be good to get a draft in the PR that I can edit if needed.

Reply:
> Would this be better in the `Ui` module instead of here? I'm honestly undecided, so just asking to get others thoughts.

@lnewson You mean that we should move EditorUi and EditorUiApi to `tinymce/modules/tinymce/src/core/main/ts/api/ui/Ui.ts`?

@tylerkelly13 Updated `tinymce.editor.ui.UI.js`. See if I understood you guys correctly.

Reply:
Good question... when I originally commented I was thinking both to be honest. Up to you though, because as I mentioned I'm not sure if we should move it or not.

Reply:
I think it makes more sense to move it